django_irods

```diff
@@ -1 +1 @@
-Subproject commit 393adf318878e21b66dec1d70f29c508ab2ba77c
+Subproject commit b02faaaf759d9c78885b24c8bdd6bd040be86844
```

Comment:
I don't believe you made any changes to django_irods submodule, so please remove this change. It is better to git add selected files rather than do a git add . to add all local changes to avoid these unintended consequences.